Comments/components/commentBrowser.js

```diff
@@ -42,12 +45,22 @@ export class CommentBrowser extends Window {
       this.initializeContainers();
       this.initializeExtents();
       this.relayoutWindow();
-      $world.addMorph(this);
+
       instance = this;
+      this.name = 'comment browser';
+      this.commentGroups = {}; // dict Morph id -> Comment group morph
+
+      this.makeVisible();
```

Comment:
Since this is done in the if and the else branch, pull it out of the conditional and put it before the return. The else block can then just be abandoned?

Reply:
Yes indeed, thanks!

Reply:
Indeed